src/server/server.js

```diff
@@ -394,4 +404,58 @@ function processPluginHtml(html, pluginId) {
     });
 }
 
+function localizeHtmlFile(htmlFile, html, lang) {
+    // Replace localized strings
+    var targetJsonFile = getI18nJsonFile(htmlFile, lang);
+    if (utils.existsSync(targetJsonFile)) {
+        try {
+            var locItems = require(targetJsonFile);
```

Comment:
I believe that this declaration should be moved out of the try, at least to avoid confusion when it is used further down.